src/System.Data.SqlClient/src/System/Data/Common/AdapterUtil.SqlClient.cs

```diff
@@ -297,6 +329,89 @@ internal static InvalidOperationException NonSequentialColumnAccess(int badCol,
             return InvalidOperation(SR.GetString(SR.ADP_NonSequentialColumnAccess, badCol.ToString(CultureInfo.InvariantCulture), currCol.ToString(CultureInfo.InvariantCulture)));
         }
 
+        internal static bool CompareInsensitiveInvariant(string strvalue, string strconst)
+        {
+            return (0 == CultureInfo.InvariantCulture.CompareInfo.Compare(strvalue, strconst, CompareOptions.IgnoreCase));
+        }
+
+        static internal Exception InvalidXmlInvalidValue(string collectionName, string columnName)
+        {
+            return Argument(SR.GetString(SR.MDF_InvalidXmlInvalidValue, collectionName, columnName));
+        }
+
+        static internal Stream GetXmlStreamFromValues(String[] values, String errorString)
+        {
+            if (values.Length != 1)
+            {
+                throw ADP.ConfigWrongNumberOfValues(errorString);
+            }
+            return ADP.GetXmlStream(values[0], errorString);
+        }
+
+        static internal ConfigurationException ConfigWrongNumberOfValues(string settingName)
+        {
+            return Configuration(SR.GetString(SR.OleDb_ConfigWrongNumberOfValues, settingName));
+        }
+        static internal ConfigurationException Configuration(string message)
+        {
+            ConfigurationException e = new ConfigurationErrorsException(message);
+            return e;
+        }
+
+        static internal Stream GetXmlStream(String value, String errorString)
+        {
+            Stream XmlStream;
+            const string config = "config\\";
+            // get location of config directory
+            string rootPath = System.Runtime.InteropServices.RuntimeEnvironment.GetRuntimeDirectory();
```

Comment:
Thanks for catching this. This doesn't seem relevant on .Net Core. On the .Net Framework, this code allowed the users to override the XML config to read, by modifying the Machine config. However this mechanism doesn't hold true anymore. I will remove this code.